catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -359,6 +369,28 @@ private NitfHeaderAttribute(String longName, String shortName,
         ATTRIBUTES.add(this);
     }
 
+    private static String convertFormat(final FileType format) {
+        if (format == FileType.NITF_TWO_ONE || format == FileType.NITF_TWO_ZERO) {
+            return NITF;
+        } else if (format == FileType.NSIF_ONE_ZERO) {
+            return NSIF;
+        }
+
+        return NITF;
```

Comment:
I think I tend to agree with @dcruver - if these fields aren't correct, you're going to have all kinds of issues. Pretty much everything else is predicated on what these fields say. The only reason to allow returning "" in my opinion would be for future proofing - in case they add another type besides NITF and NSIF in the next go-round of specs. I'm sure we'd be updating the code to support the new stuff, but fielded systems might still be able to limp along as long as the file structure stays somewhat the same. But based on the current code, we might never make it here regardless.
I've pushed a PR to the FileType in the imaging app that includes these functions. Those changes will return the actual 'text' portion of the FileType: NITF, NSIF, or UNKNOWN. Once the parser is updated to depend on the latest imaging app, this code will change to call those methods on FileType and you'll need to figure out what to do with 'UNKNOWN' - which at that point, you should probably throw the IllegalStateException since it's not legit anyway.
Since this is temporary, and we are most likely arguing about a situation that will never occur, I'm going ahead and approving.